### ISSUE TITLE
Accented characters replaced when copying and pasting

### ISSUE THREAD

**Issue by Chucky2401** [microsoft/vscode#158376]
Type: <b>Bug</b>

Steps to Reproduce:

1. Edit a file encoding with Windows 1252 with accented characters and save it
2. Open another(s) file(s) in tab
3. Close and restart VSCode
4. Waiting everything is loaded
5. Switch to the first file and paste something
6. All accented characters are replaced by �

VS Code version: Code 1.70.0 (da76f93349a72022ca4670c1b84860304616aaa2, 2022-08-04T04:38:16.462Z)
OS version: Windows_NT x64 10.0.22000
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 5825U with Radeon Graphics          (16 x 1996)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.31GB (3.28GB free)|
|Process Argv|--crash-reporter-id 7cc05f40-ee51-480b-9bb7-ed21dc13136b|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (49)</summary>

Extension|Author (truncated)|Version
---|---|---
better-comments|aar|3.0.2
increment-selection|alb|0.2.0
Bookmarks|ale|13.3.1
alignment|ann|0.3.0
cobol-language-support|Bro|1.0.2
gitignore|cod|0.9.0
vscode-markdownlint|Dav|0.47.0
githistory|don|0.6.19
gitlens|eam|12.1.2
vscode-html-css|ecm|1.13.1
LogFileHighlighter|emi|2.16.0
vscode-toggle-column-selection|eri|1.0.6
prettier-vscode|esb|9.5.0
vscode-markdown-todo|fab|1.1.1
vscode-todo-plus|fab|4.18.4
git-project-manager|fel|1.8.2
vscode-firefox-debug|fir|2.9.8
gitlab-workflow|Git|3.48.1
reg|ion|1.1.0
vscode-edit-csv|jan|0.7.2
better-shellscript-syntax|jef|1.1.0
diff-tool|jin|0.0.1
auto-comment-blocks|kev|1.0.1
eml|lei|0.4.0
sftp|lix|1.12.10
git-graph|mhu|1.30.0
vscode-language-pack-fr|MS-|1.70.8170921
csharp|ms-|1.25.0
python|ms-|2022.12.0
vscode-pylance|ms-|2022.8.20
jupyter|ms-|2022.7.1102252217
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.9
remote-ssh|ms-|0.84.0
remote-ssh-edit|ms-|0.80.0
remote-wsl|ms-|0.66.3
cpptools|ms-|1.11.5
powershell|ms-|2022.7.2
sublime-keybindings|ms-|4.0.10
sqltools|mtx|0.23.0
sqltools-driver-mssql|mtx|0.2.0
material-icon-theme|PKi|4.19.0
auto-version-increment|SBG|1.4.10
markdown-preview-enhanced|shd|0.6.3
swdc-vscode|sof|2.6.33
remote-x11-ssh|spa|1.5.0
markdowntable|Tak|0.10.1
vscode-workspace-explorer|tom|1.5.0
markdown-pdf|yza|1.4.4

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonvspyt551cf:30345471
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscorecescf:30445987
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30542924
vsaa593cf:30376535
pythonvs932:30410667
wslgetstarted:30449410
cppdebug:30492333
pylanb8912:30545647
vsclangdf:30486550
c4g48928:30535728
hb751961:30543456
dsvsc012cf:30540253
azure-dev_surveyone:30548225

```

</details>

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.70.1. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!

**Comment by Chucky2401** [comment]
I update to 1.70.1 and the issue remains.

**Comment by bpasero** [comment]
VS Code supports a wide range of file encodings. You can see the full list when you click on the lower right status bar where the encoding is shown and then choose to either reopen the file with a specific encoding or save the file with a specific encoding:

![image](https://user-images.githubusercontent.com/900690/33760971-db202986-dc07-11e7-8ca0-f8d8938cef7f.png)

There are two options how to configure the encoding for files

**Configure the encoding as a setting for all files or files of specific type**
We provide a `files.encoding` setting that you can use to set an encoding for all files. You can also configure this encoding for specific file types, e.g. like this:

```json
"[markdown]": {
    "files.encoding": "utf8"
}
```

**Configure VS Code to automatically detect the encoding when opening files**
We provide a `files.autoGuessEncoding` setting that you can enable to let VS Code detect the encoding of every file automatically. Note that this approach is known to not be accurate in all cases. This guessing of encoding is rather a heuristic that can produce wrong results (that is also why this setting is not enabled by default).

Please also check out issues that are already reported with the `files-encoding` label: https://github.com/Microsoft/vscode/issues?q=is%3Aopen+is%3Aissue+label%3Afile-encoding

**Comment by Chucky2401** [comment]
This was not help me.
I use the option to set encoding of my PowerShell script files to *windows 1252*, because as I'm French, I have accented characters, and I don't find another solution to display these characters correctly in the console.

If I set the file to any other encoding (utf8, utf16, etc.) I don't have this issue.

But, I removed my **settings.json** file, modify my settings again, and the issue is fixed now.
I don't know what was causing my issue.
